std/os/test.zig

```diff
@@ -0,0 +1,12 @@
+const std = @import("../index.zig");
+const os = std.os;
+const io = std.io;
+
+const a = std.debug.global_allocator;
+
+test "makePath, put some files in it, deleteTree" {
+    try os.makePath(a, "os_test_tmp/b/c");
+    try io.writeFile(a, "os_test_tmp/b/c/file.txt", "nonsense");
+    try io.writeFile(a, "os_test_tmp/b/file2.txt", "blah");
+    try os.deleteTree(a, "os_test_tmp");
```

Comment:
Let's add to this another line, where we try to `Dir.open` the dir we just deleted, and assert that we get `error.PathNotFound`.